cyoa/app/controllers/rounds_controller.rb

```diff
@@ -0,0 +1,16 @@
+class RoundsController < ApplicationController
+  def show
+    game = current_user.games.sort_by(&:created_at)
+    @current_game = game.last
```

Comment:
There's an ActiveRecord way to do this that's much more efficient. Can you find it?